### DISCUSSION TITLE
ERROR: CLI_ARGS:1: unused parameter

### DISCUSSION THREAD

**Discussion by FanZhangchen** [idaholab/moose#22008] [Q&A General] [answered]
Hi MOOSE team,

I am trying to follow the workshop and do step6 of the tutorials. However, when I ran the command below
`cd ~/projects/moose/tutorials/darcy-thermo_mech/step06_coupled_darcy_heat_conduction
make -j 12
../darcy_thermo_mech-opt -i step6a_coupled.i Mesh/nx=50 Mesh/ny=3 Executioner/num_steps=1 Executioner/automatic_scaling=0 -pc_type svd -pc_svd_monitor -ksp_view_pmat`,
I got some error feedback from the terminal, which are shown below
`*** ERROR ***
CLI_ARGS:1: unused parameter 'Mesh/nx'
CLI_ARGS:1: unused parameter 'Mesh/ny'
CLI_ARGS:1: unused parameter 'Mesh/nx'
CLI_ARGS:1: unused parameter 'Mesh/ny'`
Do you have any ideas on how to solve it?

Thank you so much.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Isn’t there a subblock in Mesh? Like Mesh/cmg/nx ?

Guillaume

**Reply by FanZhangchen** [reply]
Thank you for your help. There is a subblock in Mesh named gmg. The command could be run after I used Mesh/gmg/nx.
